tools/xcodeproj_shims/build-wrapper.sh

```diff
@@ -6,6 +6,10 @@ if [ $BAZEL_EXECUTION_LOG_ENABLED -gt 0 ]; then
     EXEC_LOG_OPTION="--experimental_execution_log_file=$BAZEL_BUILD_EXECUTION_LOG_FILENAME"
 fi
 
+if [ -n "$(export -p | grep BAZEL_IOS_MULTI_CPUS=)" ]; then
+    EXEC_LOG_OPTION="$EXEC_LOG_OPTION --ios_multi_cpus=$BAZEL_IOS_MULTI_CPUS"
```

Comment:
this probably doesn't belong in `$exec_log_option` -- i'd reccomend changing this to an array of args passed to bazel that we can freely append to

Reply:
Apparently I cannot read.

Reply:
Done